go/kbfs/libkbfs/favorites.go

```diff
@@ -79,12 +81,41 @@ type favReq struct {
 	ctx context.Context
 }
 
+type favoriteData struct {
+	Name         string
+	FolderType   keybase1.FolderType
+	ID           keybase1.TLFID
+	Private      bool
+	TeamID       keybase1.TeamID
+	ResetMembers []keybase1.User
+}
+
+func favoriteDataFrom(folder keybase1.Folder) favoriteData {
+	return favoriteData{
+		Name:       folder.Name,
+		FolderType: folder.FolderType,
+		// TODO: get ID??
```

Comment:
Why is there a TLF ID in `favoriteData` at all?  Doesn't seem like it's used anywhere.

Reply:
Not sure.. I removed it.